### PR TITLE
Add ExcludeCSS SKO closes #707

### LINKED ISSUE #707: Add ExcludeCSS SKO Documentation + name fixes
## Description of documentation change
Add these two Skin Objects:
- CurrentDate
- ExcludeCSS

Also:
DNNCSSEXCLUDE ->CSSEXCLUDE 
DNNCSSINCLUDE -> CSSINCLUDE 
DNNJSEXCLUDE ->JSEXCLUDE 
DNNJSINCLUDE ->JSINCLUDE 

As those are the correct names

## Existing documentation URL (if applicable)
[Provide URL to existing documentation.](https://docs.dnncommunity.org/content/tutorials/themes/theme-objects/)

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**
Closes #707

- Add page for ExcludeCss
- Changed main index page to include ExcludeCSS
- Change main index page to use correct SKO names (without DNN)
- Update CurrentDate previous page value

### CODE REVIEW

File: content/tutorials/toc.md [hunk 1]
```diff
@@ -233,6 +233,10 @@
 ###[Install Your Theme](xref:install-extension)
 ##[Theme Objects](xref:theme-objects)
 ###[Breadcrumb](xref:breadcrumb)
+###[CSSINCLUDE](xref:cssinclude)
+###[CSSEXINCLUDE](xref:cssexinclude)
```
Comment (david-poindexter) [inline content/tutorials/toc.md:237]:
```suggestion
###[CSSEXINCLUDE](xref:cssexclude)
```

File: content/tutorials/themes/theme-objects/currentdate/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
-uid: currentdate  
+uid: current date
```
Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/currentdate/index.md:2]:
```suggestion
uid: currentdate  
```
The `uid` should not have spaces in it.

File: content/tutorials/toc.md [hunk 3]
```diff
@@ -233,6 +233,10 @@
...
+###[COPYRIGHT](xref:copyright)
+###[CURRENTDATE](xref:breadcrumb)
```
Comment (david-poindexter) [inline content/tutorials/toc.md:239]:
```suggestion
###[CURRENTDATE](xref:currentdate)
```

**Review by david-poindexter — CHANGES_REQUESTED** [review]
Just a few typos corrected.  I'll merge suggestions and approve.

**Comment by david-poindexter** [comment]
@Timo-Breumelhof I edited your PR description to include the `Closes #707` syntax.  Otherwise, it would not auto-close the issue.

**Review by david-poindexter — APPROVED** [review]

**Comment by Timo-Breumelhof** [comment]
@david-poindexter thank you :-)

**Pull request merged**